android/build.gradle

```diff
@@ -2,11 +2,11 @@
 buildscript {
   repositories {
     google()
-    jcenter()
```

Comment:
Could you verify that this change is correct? Previously, this change made the module broken and fixed in #1328

Reply:
Hello @hyochan, I noticed that, thank you for fixing it last time. This time though, I have made sure everything works by re-running the new CI check for android https://github.com/dooboolab/react-native-iap/pull/1333/checks?check_run_id=2602685566 now we can move faster with more confidence